vendor/update-pp.sh

```diff
@@ -20,6 +21,8 @@ mkdir -p pp/src
 SRC=$TMP/pp
 
 cp -v $SRC/src/pp.{ml,mli} pp/src
+cp -v $SRC/test/tests.ml pp/test/tests.ml
```

Comment:
Where is tests.ml?

Btw, there's actually no need to vendor the test at all. It's enough to run the tests in ocaml-dune/pp

Reply:
We don't run it there AFAICT.

Reply:
I didn't rerun the script, let me try again.

Reply:
We won't run it here either. We don't run tests in the vendored directory. So we might as well stop copying it.

Reply:
OK I will make a PR adding some basic github actions to the fork.

Reply:
and I'll drop the tests